### PR TITLE
feat(core/inlines): support linking element-attr types

### PR DESCRIPTION
**Pull request by sidvishnoi**
Closes https://github.com/w3c/respec/issues/2705
~Blocked on https://github.com/marcoscaceres/respec.org/pull/61~ Updated on respec.org
In future, https://github.com/w3c/respec/pull/2508 might also help.

### CODE REVIEW

**Comment by sidvishnoi** [comment]
I don't have permission to merge https://github.com/marcoscaceres/respec.org/pull/61, but I have access to respec.org server, so I can deploy there if this is needed in an urgent manner. 
cc: @janiceshiu

**Comment by saschanaz** [comment]
How does bikeshed do this BTW?

**Comment by sidvishnoi** [comment]
> How does bikeshed do this BTW?

Bikeshed: `<{element/attr}>`
ReSpec: `[^element/attr^]` (as we wanted valid HTML markup)
https://tabatkins.github.io/bikeshed/#autolink-wrappers

**Comment by janiceshiu** [comment]
@sidvishnoi important but not urgent imho. right now this prevents me from using xref to link any HTML attributes references in other specs.

i can always put this on the backburner and raise another PR to add the xref references at a later time.

**Comment by sidvishnoi** [comment]
I'm gonna update the server to incorporate https://github.com/marcoscaceres/respec.org/pull/61.

File: tests/spec/core/xref-spec.js [hunk 1]
```diff
@@ -237,14 +237,14 @@ describe("Core — xref", () => {
   it("shows error if cannot resolve by data-cite", async () => {
     const body = `
       <section data-cite="svg">
-        <p><a id="link">symbol</a> twice in svg spec.</p>
+        <p id="test">[^symbol^] twice in svg spec.</p>
```
Comment (sidvishnoi) [inline tests/spec/core/xref-spec.js]: We earlier treated elements as concepts. Now, element's have their own category (element types). So, updated test.

**Pull request merged**